t/plugin/kafka-logger.t

```diff
@@ -592,3 +592,44 @@ qr/partition_id: 2/]
 [qr/partition_id: 1/,
 qr/partition_id: 0/,
 qr/partition_id: 2/]
+
+
+
+=== TEST 20: user sasl create producer
+--- extra_init_by_lua
+local producer = require("resty.kafka.producer")
+producer.send = function()
+    return true
+end
+--- config
+    location /t {
+        content_by_lua_block {
+            local producer = require "resty.kafka.producer"
```

Comment:
We should test the kafka-logger plugin directly like other tests, instead of using the producer outside of the plugin.